spec/requests/api/status_spec.rb

```diff
@@ -27,5 +43,31 @@
       expect(response.body.include?(project.name)).to be == true
       expect(response.body.include?('Django')).to be == true
     end
+
+    it "renders empty maintenance stats if they don't exist" do
+      post "/api/check", params: {api_key: user.api_key, projects: [{name: project_django.name, platform: project_django.platform}] }
+      expect(response).to have_http_status(:success)
+      expect(response.content_type).to eq('application/json')
+      expect(response.body.include?(project_django.name)).to be == true
+
+      json_response = JSON.parse(response.body)
+      expect(json_response.first["repository_maintenance_stats"].length).to be 0
+    end
+
+    context "with normal API key" do
+      before do
+        user.current_api_key.update_attribute(:is_internal, false)
```

Comment:
Maybe have two different users, one internal and one not?

Reply:
I can do that. It will still look very similar. There isn't an easy way to designate a user or key as internal through the test factories.